### ISSUE TITLE
Accept HTML only custom element in moduleId RouteConfig

### ISSUE THREAD

**Issue by AbraaoAlves** [aurelia/templating-router#49]
**I'm submitting a feature request**

**Expected/desired behavior:**
* **What is the expected behavior?**
The moduleId should work not just ViewModels, but also HTML only custom elements.

Like tag `<require from="my-component.html">`, that accept viewModels and Views, moduleId in RouteConfig should accept string like this:

``` javascript
    config.map([
       { route: ['', '/'], moduleId: 'no-selection.html',  title: 'Select'},
       { route: 'contacts/:id',  moduleId: 'contact-detail',  name:'contacts'}  
    ]);
```

**What is the motivation / use case for changing the behavior?**
Less is MORE. Write empty class to just route works, should not be the Aurelia way.

**Comment by EisenbergEffect** [comment]
Closing this as a duplicate. We have a feature tracked to enable "view only" routes. Thanks!

**Comment by AbraaoAlves** [comment]
@EisenbergEffect  any link?

**Comment by AbraaoAlves** [comment]
I don't find any feature like this. Yes is there #25, but is about layout view-only no page view-only.

Take app-contacts app and put this code on app.js:

``` javascript
config.map([
   { route: ['', '/'], **moduleId: 'no-selection.html',**  title: 'Select'},
   { route: 'contacts/:id',  moduleId: 'contact-detail',  name:'contacts'}  
]);
```
The aurelia-template-route don't work with moduleId path. But if I make little change on [`route-loader.js`](https://github.com/abraaoalves/templating-router/blob/master/src/route-loader.js#L16) ... It works, not more empty class!! 


I want know if a way to do this, without change source code?

**Comment by carusology** [comment]
I also ran into this limitation. As far I as I can tell, we are required to create an empty view-model to back the view. @EisenbergEffect, can you give us a link to whatever issue is tracking this feature request? The issue @AbraaoAlves referenced, #25, does not appear to be related.

**Comment by EisenbergEffect** [comment]
I thought we had a feature for this somewhere. Perhaps, I was wrong? I'll re-open this.

**Comment by carusology** [comment]
@EisenbergEffect k, thanks.

When I attempted to get this to work within the current architecture, I used the `my-view.html` style of syntax in [that is referenced in Aurelia Hub for HTML-Only custom elements](http://aurelia.io/hub.html#/doc/article/aurelia/templating/latest/templating-custom-elements/2), but put it in my route config, like so:

```
export class App {

  configureRouter(config, router) {
    this.router = router;
    config.title = 'Aurelia';

    config.map([
      { route: ['', 'home'], name: 'home', moduleId: './my-view.html' }
    ]);
  }
}
```

However, this gets me message `"Uncaught (in promise) Error: Cannot find module './my-view'."`. Once I added a blank view-model (which, to be fair, is a single line of code), it worked fine.

```
// filename: my-view.ts
export default class MyView {}
```

This `*.html` suffix is the exact approach @AbraaoAlves suggested in [his pull request](https://github.com/aurelia/templating-router/pull/50). Did you follow the same deductive path, @AbraaoAlves?

I keep expecting custom elements and views to behave in a similar fashion, but that expectation is consistently [wrong](https://github.com/aurelia/templating/issues/415).

**Comment by AbraaoAlves** [comment]
That's it @carusology, clear like water.
Thanks!

**Comment by gheoan** [comment]
@EisenbergEffect, @AbraaoAlves I think this can be closed since #50 was merged.

**Comment by nick-janda** [comment]
@AbraaoAlves  I see there was a pull request to fix this, but it seems that it only works when the html-only component is in the root where app.js is, but doesn't appear to work when in a child folder.  Is there a reason why a html-only custom element in a child folder won't work here?
